WebService/v1/Controllers/StatusController.cs

```diff
@@ -17,20 +18,23 @@ namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Controller
     public sealed class StatusController : Controller
     {
         private readonly IDevices devices;
-        private IStorageAdapterClient storage;
+        private readonly IStorageAdapterClient storage;
         private readonly ISimulations simulations;
         private readonly ILogger log;
+        private readonly IServicesConfig servicesConfig;
 
         public StatusController(
             IDevices devices,
             IStorageAdapterClient storage,
             ISimulations simulations,
-            ILogger logger)
+            ILogger logger,
+            IServicesConfig servicesConfig)
         {
             this.devices = devices;
             this.storage = storage;
             this.simulations = simulations;
             this.log = logger;
+            this.servicesConfig = servicesConfig;
         }
 
         [HttpGet]
```

Comment:
Split method into sub-methods for readability/debugging.

Reply:
:+1: